build-aux/deb/init.d/opentsdb

```diff
@@ -30,10 +30,6 @@ MAX_OPEN_FILES=65535
 # The first existing directory is used for JAVA_HOME
 # (if JAVA_HOME is not defined in $DEFAULT)
 JDK_DIRS="/usr/lib/jvm/java-8-oracle \
-   /usr/lib/jvm/java-7-oracle /usr/lib/jvm/java-7-openjdk \
-   /usr/lib/jvm/java-7-openjdk-amd64/ /usr/lib/jvm/java-7-openjdk-i386/ \
-   /usr/lib/jvm/java-6-sun /usr/lib/jvm/java-6-openjdk \
```

Comment:
I don't mind removing 6 for 2.4, but we should leave 7 as it's not fully deprecated yet and a number of folks haven't moved to 8 yet.